### PR TITLE
VACMS-3670 - Update clp ticket items in ticket list

### PR DESCRIPTION
**Pull request by ethanteague**
## Depends on / Requires merge first
https://github.com/department-of-veterans-affairs/va.gov-cms/pull/3666

## Description
See #3670 

## Testing done
Visual / Behat

## QA steps
This is going to require manual qa by @kevwalsh 

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Core Application Team`
- [x] `Product Support Team`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change. 
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: docroot/modules/custom/va_gov_backend/va_gov_backend.module [hunk 1]
```diff
@@ -1168,6 +1224,46 @@ function va_gov_backend_entity_bundle_field_info_alter(&$fields, EntityTypeInter
...
+        'readable' => 'Stories',
+        'min' => 1,
+        'max' => 2,
+      ]);
+    }
+    // Add range check on spotlight panel.
+    if (isset($fields['field_clp_spotlight_link_teasers'])) {
+      $fields['field_clp_spotlight_link_teasers']->addConstraint('RequiredParagraph', [
+        'toggle' => 'field_clp_spotlight_panel',
+        'readable' => 'Additional links',
```
Comment (kevwalsh) [inline docroot/modules/custom/va_gov_backend/va_gov_backend.module:1251]: Not sure where "Additional links" came from.  How about "Links with summaries", for now.

File: config/sync/field.field.node.campaign_landing_page.field_clp_resources.yml [hunk 2]
```diff
@@ -17,7 +17,7 @@ field_name: field_clp_resources
 entity_type: node
 bundle: campaign_landing_page
 label: 'Media resources'
-description: ''
+description: 'You must add resources to the <a href="/media/add/document_external">media library</a> before you will be able to select them here.'
 required: false
```
Comment (kevwalsh) [inline config/sync/field.field.node.campaign_landing_page.field_clp_resources.yml:21]: not sure what value should be here for required, but it should require between 1 and 3, and it should have an asterisk for now to show that it's required

Comment (kevwalsh) [reply config/sync/field.field.node.campaign_landing_page.field_clp_resources.yml:21]: Create_Campaign_Landing_Page___VA_gov_CMS

**Review by kevwalsh — CHANGES_REQUESTED** [review]
@ethanteague  couple little things

File: config/sync/field.field.node.campaign_landing_page.field_clp_resources.yml (hunk 2 above)
Comment (ethanteague) [reply config/sync/field.field.node.campaign_landing_page.field_clp_resources.yml:21]: Yep. That logic is in place (happening in the module file). My js missed adding the asterisk - good catch, @kevwalsh

**Comment by ethanteague** [comment]
Closing in favor of https://github.com/department-of-veterans-affairs/va.gov-cms/pull/3714

**Pull request closed without merging**